packages/fx-core/src/plugins/resource/apiconnector/plugin.ts

```diff
@@ -426,4 +443,26 @@ export class ApiConnectorImpl {
     node.addChild(queryKeyNameQuestionNode);
     return node;
   }
+
+  public exactTelemetryProperties(config: ApiConnectorConfiguration): { [key: string]: string } {
+    const properties = {
+      [Telemetry.properties.authType]: config.AuthConfig.AuthType.toString(),
+    };
+    switch (config.AuthConfig.AuthType) {
+      case AuthType.AAD:
+        const aadAuthConfig = config.AuthConfig as AADAuthConfig;
+        properties[Telemetry.properties.reuseTeamsApp] = aadAuthConfig.ReuseTeamsApp
+          ? Telemetry.valueYes
+          : Telemetry.valueNo;
+        break;
+      case AuthType.APIKEY:
+        const authConfig = config.AuthConfig as APIKeyAuthConfig;
+        properties[Telemetry.properties.keyInHeader] =
```

Comment:
can we just add a api key location property who's value might be query parameter or request header? keyInHeader = false may not be enough to tell people where is the key